src/local-build.sh

```diff
@@ -1,115 +0,0 @@
-#!/bin/bash
-
-# Local build script for front ends
-# This script is used to build the front end locally
-# Use this to test the frontend build system locally when making changes
-
-# Usage:
```

Comment:
@Victoremepunto @adamrdrew do we have these steps published somewhere else? I found them useful while building frontend images locally for patternfly 6 (on ephemeral). 

I know they are probably different now that we use the same entry point locally and in the pipeline, but we should publish the steps somewhere (inscope, or doc in bonfire?)